docs/smart-contracts/quickstart.md

```diff
@@ -171,6 +219,42 @@ The `Hello World` smart contract stores a greeting in its state, and exposes two
 
   </TabItem>
 
+  <TabItem value="py" label="🐍 Python">
+
+```python
+from near_sdk_py import Contract, view, call, init
+
+class GreetingContract(Contract):
+    """
+    A simple greeting contract that stores and returns a message.
+    """
+    
+    @init
+    def initialize(self, default_message="Hello, NEAR world!"):
+        """
+        Initialize the contract with a default greeting message.
+        """
+        self.storage["greeting"] = default_message
+        return {"success": True}
+    
+    @call
+    def set_greeting(self, message: str):
+        """
+        Change the greeting message.
+        """
+        self.storage["greeting"] = message
+        self.log_info(f"Saving greeting: {message}")
+        return {"success": True}
+    
+    @view
+    def get_greeting(self):
+        """
+        Retrieve the current greeting message.
+        """
+        return self.storage.get("greeting", "Hello, NEAR world!")
+```
+
```

Comment:
can we upload all this code to a GitHub repo, so it can be linked directly to the original source code? (as we are currently doing for JS and Rust)

also it needs to be mentioned that the file name must be `greeting_contract.py`. Right now I think people will end up assuming the code goes into `main.py` and the following tests and build won't work.

Reply:
@r-near  I have created a PR on the `hello-near-examples` repository where the Rust and JS example code is stored, adding the Python example contract, so everything remains consistent: https://github.com/near-examples/hello-near-examples/pull/28

Please feel free to review and suggest changes to that PR, and then we can link to those python files from our docs.